OPERATOR.rst

```diff
@@ -156,28 +156,50 @@ label from blocking PR. Last, remove the blocking relationship.
 Upgrading GitLab
 ^^^^^^^^^^^^^^^^
 
-Occasionally it falls on the operator to upgrade the Azul GitLab instance. If
-the current major version is ``n`` and the latest available major version is
-greater than ``n+1`` (i.e. upgrading directly to the latest version would skip
-one or more major versions) then multiple successive upgrades must be made, such
-that no upgrade skips a major version. For example, if the current version is
-13.x.y and the latest available version is 15.x.y, then one would first upgrade
-to 14.x.y and then repeat the process to upgrade to 15.x.y.
-
-Before any changes are applied, stop the instance (do not terminate) and create
-a snapshot of its EBS volume. Edit ``terraform/gitlab/gitlab.tf.json.template.py``,
-updating the versions of the docker images for ``gitlab-ce`` and
-``gitlab-runner``. Then run::
+Operators must check for updates to GitLab on a daily basis. Updates to Gitlab
+can be quickly checked at the ``/help`` endpoint for `GitLab dev`_ and
+`GitLab prod`_. If there are updates available, find the new release for the
+`GitLab Docker image`_ and update the `Gitlab Terraform`_ file. When updating
+the GitLab instance, check if there are applicable updates to the
+`GitLab runner image`_. Before starting the update process check the
+`GitLab release notes`_ for upgrading instructions. When upgrading across
+multiple GitLab versions, follow the prescribed GitLab `upgrade path`_.
+
+Before any changes are applied, checkout a new branch with `github/develop` as
+its start-point, run::
+
+	git fetch github
+	git checkout -b gitlab/<yyy-mm-dd> github/develop
```

Comment:
Developers should know how to create a branch from `develop` so this is overly verbose. Besides, the origin name (`github` in your case may be different for each developer. Be less detailed, but mention the branch name.

Reply:
@hannes-ucsc 
the origin name should not be different as we standardized it for operators. 
https://github.com/DataBiosphere/azul/blob/develop/OPERATOR.rst#getting-started-as-operator

Reply:
Got it. Might still be more concise to say "Create a branch called ... from `develop`." There still is an imbalance here between sections that state the obvious and sections that elide the not-so-obvious.

BTW, you don't need to @-mention me in replies. I get notified and see them regardless.

Reply:
> Should we expect developers to keep branches updated as part of their development workflows?

Yes. Keeping up with the remote is an obvious and therefore implied requirement. Everything breaks if developers or operators use stale branches.